fastai/train.py

```diff
@@ -100,12 +100,13 @@ def clip_grad(learn:Learner, clip:float=0.1)->Learner:
 
 class ClassificationInterpretation():
     "Interpretation methods for classification models."
-    def __init__(self, data:DataBunch, probs:Tensor, y_true:Tensor, losses:Tensor, ds_type:DatasetType=DatasetType.Valid):
-        self.data,self.probs,self.y_true,self.losses,self.ds_type = data,probs,y_true,losses,ds_type
+    def __init__(self, data:DataBunch, learn:Learner, probs:Tensor, y_true:Tensor, losses:Tensor, ds_type:DatasetType=DatasetType.Valid):
```

Comment:
So we need to had the `Learner` for this to work, which is fine. Let's remove `DataBunch` then since we can access it with `learn.data`. This needs a bit of adjustment after (and in the `from_learner` method, don't forget to pass this `Learner`), so let me know if you'd rather have me doing it.